packages/create-cloudflare/src/__tests__/git.test.ts

```diff
@@ -163,11 +162,23 @@ describe("git helpers", () => {
 				["git", "init", "--initial-branch", "main"],
 				expect.any(Object),
 			);
+			expect(ctx.gitRepoAlreadyExisted).toBe(false);
 			expect(ctx.args.git).toBe(true);
 		});
 
 		test("git not installed", async () => {
```

Comment:
Comment:

> "git not installed" (This particular instance is not updated in this PR)

A guideline I find very useful (from Google) is to describe both the context and the outcome for test.

i.e. here it could help understand:

```ts
			expect(updateStatus).not.toHaveBeenCalled();
			expect(processArgument).not.toHaveBeenCalled();
```

Reply:
Yes I agree.

Reply:
323e23694d7c483868c4f17085c8ff9092597efb